handbook/engineering/campaigns/goals.md

```diff
@@ -1,66 +1,69 @@
-# Campaigns goals and priorities
+# Campaigns goals
 
-## Goals
+The campaigns team is building the best solution available for creating and managing large scale changes. To do so we will focus on the following objectives:
 
-### Build a base of 10 customers who have created 80 campaigns and made 4000 changesets in Q4
+## Grow adoption of campaigns 
 
-We want to get a solid set of customers who regularly use and rely on campaigns. 
+We need to grow usage of campaigns to understand our customer use cases and inform our roadmap.
 
-What will get us there?
+How do we do this:
+  * Proactive customer outreach
+  * Continual user validation
+  * Strive to create a frictionless initial experience
+  * Provide amazing support
```

Comment:
These all sound pretty buzzwordy to me and are not actionable to me.

Do I, as an engineer, do the proactive customer outreach?

What does "Continual user validation" mean besides releasing new features and getting feedback? Could we make this more concrete by saying "Ship one new thing-to-validate in each sprint"?

"Strive to create a frictionless initial experience" — sure, who doesn't? But it seems to be more of a goal, not a "how".

Amazing support — well, that can mean anything :)